### DISCUSSION TITLE
Whisper TFlite integrated within OpenVoiceOS

### DISCUSSION THREAD

**Discussion by j1nx** [usefulsensors/openai-whisper#17] [Show and tell]
The openai-whisper tensorflow lite runtime model is integrated as a STT plugin within OpenVoiceOS to have local STT on an embedded device without to much delay (still tweaking the best options and way forward with the tflite_runtime delegates)

https://github.com/OpenVoiceOS/ovos-stt-plugin-whisper-tflite

A little quick and dirty showcase video.
https://www.youtube.com/watch?v=Aor6CFkcWzU

**Comment by nyadla-sys** [comment]
@j1nx It's great to see that Whisper TFLite models are being utilized.
